spec/features/staff/users_can_create_a_fund_level_activity_spec.rb

```diff
@@ -130,11 +130,18 @@
         fill_in "activity[planned_end_date(2i)]", with: 12
         fill_in "activity[planned_end_date(1i)]", with: 2010
         click_button I18n.t("form.activity.submit")
+        expect(page).to have_content I18n.t("page_title.activity_form.show.geography")
+
+        click_button I18n.t("form.activity.submit")
+        expect(page).to have_content "can't be blank"
+
+        choose "Region"
+        click_button I18n.t("form.activity.submit")
         expect(page).to have_content I18n.t("page_title.activity_form.show.region")
 
-        # Region has a default and can't be set to blank so we skip
-        select "Developing countries, unspecified", from: "activity[recipient_region]"
+        # region has the default value already selected
         click_button I18n.t("form.activity.submit")
+
```

Comment:
Can we go back to geography and test country too?

Reply:
I feel like I covered that here:

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/feature/303-activities-collect-recipient-country/spec/features/staff/users_can_manage_activity_geography_spec.rb